tests/unit/field/test_text_field.py

```diff
@@ -134,6 +136,60 @@ def test_build_vocab_empty():
     assert field.vocab == vocab
 
 
+def test_build_vocab_setup_all_embeddings():
+    """
+    This test shows that all fields in the embeddings will be included.
+
+    In embeddings and data:
+        blue
+        green
+        yellow
+    In embeddings only:
+        purple
+        gold
+    In data only:
+        white
+
+    Expected vocab:
+        blue
+        green
+        yellow
+        purple
+        gold
+        white
+    """
+
+    field = TextField(
+        embeddings='not-a-real-embedding',
+        setup_all_embeddings=True,
+    )
+
+    embedding_model = MagicMock()
+    embedding_model.vocab = {
+        'blue': torch.rand(5),
+        'green': torch.rand(5),
+        'yellow': torch.rand(5),
+        'purple': torch.rand(5),
+        'gold': torch.rand(5),
```

Comment:
Also, when building the embeddings, lets fix the values to avoid flakey tests in the future.

Reply:
How could the tests be flakey